internal/pkg/cli/dialog/use_template.go

```diff
@@ -12,11 +12,11 @@ import (
 
 type contextKey string
 
-func (p *Dialogs) AskUseTemplateOptions(inputs input.Inputs) (results map[string]interface{}, err error) {
+func (p *Dialogs) AskUseTemplateOptions(inputs *input.Inputs) (results map[string]interface{}, err error) {
 	results = make(map[string]interface{})
 	ctx := context.Background()
 	errors := utils.NewMultiError()
-	for _, i := range inputs {
+	for _, i := range inputs.All() {
```

Comment:
z Inputs slice som spravil struct, ktora obsahuje slice.
Tak sa s tym bude jednoduchsie v kode pracovat.
Definovat metody priamo nad `slice`, ktore ju maju modifikovat, by bolo problematicke.
Musel by sa odovzdavat pointer na slice, a pomocou struktury je to lepsie.